src/Microsoft.AspNetCore.Server.Kestrel.Core/Internal/Infrastructure/KestrelTrace.cs

```diff
@@ -30,6 +30,9 @@ public class KestrelTrace : IKestrelTrace
         private static readonly Action<ILogger, string, Exception> _connectionDisconnect =
             LoggerMessage.Define<string>(LogLevel.Debug, 10, @"Connection id ""{ConnectionId}"" disconnecting.");
 
+        private static readonly Action<ILogger, string, Exception> _connectionRejected =
+            LoggerMessage.Define<string>(LogLevel.Warning, 11, @"Connection id ""{ConnectionId}"" rejected.");
```

Comment:
Soliciting feedback. Is `Warning` too high? Should this be debug or info instead?